example/lib/main.dart

```diff
@@ -10,9 +10,10 @@ void main() {
 
   dashbook.storiesOf('RaisedButton').decorator(CenterDecorator()).add(
         'default',
-        (ctx) => RaisedButton(
+        (ctx) => ElevatedButton(
           child: Text(
-            ctx.listProperty("Label", "Ok", ["Ok", "Cancel", "Other label"]),
+            ctx.listProperty("Label", "Ok", ["Ok", "Cancel", "Other label"]) ??
+                "",
```

Comment:
**question:** why does it need the default? shouldn't this not be null?

Reply:
Nope! The `listProperty` method returns `T?` and there you need a non-nullable value 😄